### PR TITLE
add config-switch to disable oqrs-feature

### PR DESCRIPTION
**Pull request by dg0tm**
It does the same as #450 but for the OQRS feature.

### CODE REVIEW

**Comment by HB9HIL** [comment]
Hello @dg0tm 

Could you please edit this PR and add a Description? Why do we need this option. What did you change and are there any special cases we should take care off?

File: application/config/config.sample.php [hunk 1]
```diff
@@ -695,3 +695,6 @@
 
 # Disables QSL-Image-Feature if true. if non-existant or false the QSL-Feature is enabled
 $config['disable_qsl'] = false;
+
+# Disables OQRS-Feature if true. if non-existant or false the OQRS-Feature is enabled
```
Comment (HB9HIL) [inline application/config/config.sample.php:699]: This config option should be in `install/config/config.php` aswell

**Review by HB9HIL — CHANGES_REQUESTED** [review]

File: application/views/interface_assets/header.php [hunk 2]
```diff
@@ -394,10 +394,12 @@ function handleKeyPress(event) {
 								}
 
 								$oqrs_requests = $this->oqrs_model->oqrs_requests($location_list);
+								if ($oqrs_requests > 0) {
```
Comment (HB9HIL) [inline application/views/interface_assets/header.php:397]: I'm not happy with this one. If OQRS is enabled we should show the menu entry for it. So the user knows where to find the requests. Also this does not work as you wrote it. 

```suggestion
							
```

**Comment by HB9HIL** [comment]
Another one noticed while testing.

If a user already had OQRS enabled for a station location it will show the OQRS menu tab in public view. 
You can also hide this based on the config option. This would be more elegant then just redirect to dashboard/user login

Check `application/views/visitor/layout/header.php` Line `75`

File: application/views/options/oqrs.php [hunk 3]
```diff
@@ -66,4 +67,5 @@
 		</div>
 	</div>
 
-</div>
\ No newline at end of file
+</div>
+<?php } >
```
Comment (HB9HIL) [inline application/views/options/oqrs.php:71]: incomplete PHP closing

```suggestion
<?php } ?>
```

File: application/views/interface_assets/header.php (hunk 2 above)
Comment (dg0tm) [reply application/views/interface_assets/header.php:397]: okay, what's the point of a menu item that shows nothing but a empty page?
and as you can see, it's an optional commit in this PR.

grafik

Comment (HB9HIL) [reply application/views/interface_assets/header.php:397]: We just discussed that in team. 

We don't want to hide this menu entry if OQRS is enabled, even if there are no requests yet. 
You can hide the menu entry if OQRS is disabled.

Comment (HB9HIL) [reply application/views/interface_assets/header.php:397]: > I am against hiding the menu item. It counts for requests with state < 2, that means menu item will only show when you have new requests. What if you want to check something on old requests?

It shows the menu entry anyway but it shows a small number with new requests if there are more then 2 new

image

Imho it should stay like this. Just hide it if OQRS is disabled @dg0tm

Comment (HB9HIL) [reply application/views/interface_assets/header.php:397]: image

**Review by HB9HIL — APPROVED** [review]
Looks good to me now

**Pull request merged**